source/_integrations/flux.markdown

```diff
@@ -8,7 +8,9 @@ ha_quality_scale: internal
 ha_domain: flux
 ---
 
-The `flux` switch platform will change the temperature of your lights similar to the way flux works on your computer, using circadian rhythm. They will be bright during the day, and gradually fade to a red/orange at night. The `flux` switch restores its last state after startup.
+The `flux` switch platform will change the temperature of your lights similar to the way flux works on your computer, using circadian rhythm. 
+They will be bright during the day, and gradually fade to a red/orange at night. See more on the [f.lux web page](https://justgetflux.com/).
```

Comment:
This might be confusing since this integration has absolutely nothing to do with the software offered on the linked page.

Reply:
The intro states, " similar to the way flux works on your computer", so I figured I would add a link to help people read about what it actually does. The software is different, but the principle is described.

Reply:
So, maybe we should describe the principle ourselves?

Reply:
But this version is a copy of a hue plugin, which is directly saying it is an implementation of f.lux, why not give credits to the original original of the f.lux software?

Reply:
This is not giving credit, this is sending people to another software package for context. Those are two different things, and the initial concern I raised. (So back at square 1)

Reply:
Okay,  i was trying to help by expanding with a link to the original software and explanation which this plugin is inspired by,  but you can just reject the PR, and I will find elsewhere to contribute.

Reply:
Think that the key question is, why is there a flux integration without a connection with the flux software. Is it actually correct that it bears the name flux? Or would it be better to integrate this integration in terms of features with an emerging integration that has the same functions? Like #14877

Reply:
> Okay, i was trying to help by expanding with a link to the original software and explanation which this plugin is inspired by, but you can just reject the PR, and I will find elsewhere to contribute.

It has nothing to do with that software. That is the whole point.

As you wish, closing.